src/ngraph/runtime/executable.hpp

```diff
@@ -80,32 +80,46 @@ class NGRAPH_API ngraph::runtime::Executable
     /// \brief Create an input Tensor
     /// \param input_index The index position in the input Parameter vector. This would be the same
     /// order of Parameters passed into the inputs in the call() method.
+    /// \param memory_pointer A pointer to a buffer used for this tensor. The size of the buffer
```

Comment:
The default value is nullptr but nowhere is it described what happens when a nullptr is passed in. Is the implementor of this this method supposed to allocate a buffer if none is passed in? Should I throw an error with a nullptr?
These comments apply to all docstrings.

Reply:
will remove nullptr, and make new API without default nullptr.